src/fontra/views/applicationsettings/panel-shortcuts.js

```diff
@@ -0,0 +1,659 @@
+  background-color: var(--ui-element-background-color);
+  border-radius: 0.5em;
+  padding: 1em;
+}
+.fontra-ui-shortcuts-panel-header {
+  font-weight: bold;
+}
+.fontra-ui-shortcuts-panel-buttons {
+  display: grid;
+  grid-template-columns: max-content max-content max-content;
+  gap: 1em;
+}
+`);
+
+export class ShortCutsPanel extends BaseInfoPanel {
+  static title = "shortcuts.title";
+  static id = "shortcuts-panel";
+
+  async setupUI() {
+    this.panelElement.innerHTML = "";
+
+    this.panelElement.style = "gap: 1em;";
+
+    const containerButtons = html.div(
+      { class: "fontra-ui-shortcuts-panel-buttons" },
+      []
+    );
+    containerButtons.appendChild(
+      html.input({
+        type: "button",
+        style: `justify-self: start;`,
+        value: "Reset all shortcuts",
+        onclick: (event) => this.resetToDefault(),
+      })
+    );
+
+    containerButtons.appendChild(
+      html.input({
+        type: "button",
+        style: `justify-self: start;`,
+        value: "Export shortcuts",
+        onclick: (event) => this.exportShortCuts(),
+      })
+    );
+
+    containerButtons.appendChild(
+      html.input({
+        type: "button",
+        style: `justify-self: start;`,
+        value: "Import shortcuts",
+        onclick: (event) => this.importShortCuts(),
+      })
+    );
+
+    this.panelElement.appendChild(containerButtons);
+    for (const [topicKey, actionIdentifiers] of Object.entries(getShortCutsGrouped())) {
+      const container = html.div({ class: "fontra-ui-shortcuts-panel" }, []);
+      container.appendChild(
+        html.createDomElement("div", {
+          class: "fontra-ui-shortcuts-panel-header",
+          innerHTML: translate(topicKey),
+        })
+      );
+      for (const actionIdentifier of actionIdentifiers) {
+        container.appendChild(new ShortCutElement(actionIdentifier));
+      }
+      this.panelElement.appendChild(container);
+    }
+  }
+
+  async resetToDefault() {
+    const result = await dialog(
+      "Reset to default",
+      "Are you sure you want to reset all shortcuts to their default settings?",
+      [
+        { title: translate("dialog.cancel"), isCancelButton: true },
+        { title: "Okay", isDefaultButton: true },
+      ]
+    );
+    if (!result) {
+      return;
+    }
+    for (const actionIdentifier of getActionIdentifiers()) {
+      setCustomShortCuts(actionIdentifier, null);
+    }
+    location.reload();
+  }
+
+  async exportShortCuts() {
+    // Only export custom shortcuts,
+    // because default shortcuts are already in the code.
+    const shortCutsDataCustom = {};
+    for (const actionIdentifier of getActionIdentifiers()) {
+      const actionInfo = getActionInfo(actionIdentifier);
+      const shortCuts = actionInfo.customShortCuts;
+      if (!shortCuts) {
+        continue;
+      }
+      shortCutsDataCustom[actionIdentifier] = shortCuts;
+    }
+    const data = JSON.stringify(shortCutsDataCustom);
```

Comment:
Use indentation of 2 spaces:
```suggestion
    const data = JSON.stringify(shortCutsDataCustom, null, 2);
```